garrysmod/lua/includes/extensions/util.lua

```diff
@@ -126,8 +126,8 @@ function util.TypeToString( v )
 	local t = type( v )

```

Comment:
Change this to use ``TypeID`` instead as it's more efficient to do numerical comparisons than string ones. Your enums here will be ``TYPE_VECTOR``, ``TYPE_ANGLE``, and ``TYPE_NUMBER``. You can also remove the string.lower call right below this once you change it.

Reply:
On it ...

Reply:
Hello, [changed here](https://github.com/Facepunch/garrysmod/pull/1622/commits/edffb502000b74e8b78392859d0ed62bef5e0254). Actually the time is relatively the same as using `if-elseif-else-end` construct.